WLogger/Application/AppDelegate.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AppDelegate.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/3/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+        window = UIWindow(frame: UIScreen.mainScreen().bounds)
+        
+        window!.rootViewController = getRootViewController()
```

Comment:
OK, what is the best practice about this (in a general case, when we actually don't know about the optional status)?

Doing this:

```        
if let unwrapped_window = window {
  unwrapped_window.rootViewController = getRootViewController()
  unwrapped_window.makeKeyAndVisible()
}
```

Or just:

```
window?.rootViewController = getRootViewController()
window?.makeKeyAndVisible()
```